cli/init/print_tarantoolctl_cfg.lua

```diff
@@ -10,23 +12,12 @@ if default_cfg == nil then
     os.exit(1)
 end
 
--- Data/lib dir discovery.
-local lib_dir = default_cfg.wal_dir
-for _, data_dir in pairs({default_cfg.memtx_dir, default_cfg.vinyl_dir, default_cfg.snap_dir}) do
-    if lib_dir == nil then
-        if data_dir ~= nil then
-            lib_dir = data_dir
-        end
-    else
-        if data_dir ~= nil and lib_dir ~= data_dir then
-            io.stderr:write("Unable to identify data directory from taractoolctl config. " ..
-            "There is uncertainty between " .. lib_dir .. " and " .. data_dir .. "\n")
-            os.exit(1)
-        end
-    end
-end
+local d = {}
```

Comment:
Please do not use names that are one character long. They are inconvenient to use for future navigation and do not describe the object.

Reply:
Renamed.